src/fable/Fable.Core/ts/Serialize.ts

```diff
@@ -39,7 +39,8 @@ export function toJson(o: any): string {
           return v.Case;
         }
         else if (v.Fields.length === 1) {
-          return { [v.Case]: v.Fields[0] };
+          const fieldValue = v.Fields[0];
+          return { [v.Case]: typeof fieldValue === 'undefined' ? null : fieldValue };
```

Comment:
Maybe in the future a maintainer could wonder why this is necessary. Could you please add a comment or maybe a reference to this PR? (e.g. `// See #611). Thanks!